packages/angular-test-app/src/preview-examples/select-ng-model.ts

```diff
@@ -10,9 +10,9 @@
 import { Component } from '@angular/core';
 
 @Component({
-  selector: 'app-example',
+  selector: 'app-example-ng-model',
   template: `
-    <ix-select name="ix-select-control" [(ngModel)]="value">
+    <ix-select [(ngModel)]="value">
```

Comment:
Why removing name

Reply:
To be consistant with the other examples.